plugin/cuda/README.md

```
@@ -0,0 +1,16 @@
+======================
+
+This is plugin of DMTCP to checkpoint CUDA programs.
+This plugin uses CUDA's checkpoint/restart API introduced
+in CUDA 12.4.
+
+## Compile
+```
+make
+```
+
+## Run
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)